### PR TITLE
chore: remove nanoid and react-native-redash to clean up some build issues

### PR DESCRIPTION
**Pull request by natew**
redash especially causes build issues, bit cleaned up nanoid while I was there.

### CODE REVIEW

**Comment by natew** [comment]
Fixes https://github.com/gorhom/react-native-bottom-sheet/issues/1020

File: package.json [hunk 1]
```diff
@@ -39,10 +39,7 @@
     "bootstrap": "yarn install && yarn example"
   },
   "dependencies": {
-    "@gorhom/portal": "1.0.13",
-    "invariant": "^2.2.4",
```
Comment (gorhom) [inline package.json:43]: `invariant` is needed, till it get replaced with `proptypes`

**Review by gorhom — CHANGES_REQUESTED** [review]
Thanks @natew for submitting this PR, all looks fine except the removal of `invariant` package, it is still needed for now

**Comment by hirbod** [comment]
Anything I can do to get this merged? :)

**Review by gorhom — APPROVED** [review]

**Comment by gorhom** [comment]
thanks @natew for submitting this PR !

**Pull request merged**